README.md

```diff
@@ -10,6 +10,6 @@ Currently, our dockerfile creates an image for a container that runs the basic f
 1. Run `docker build -t flood .`. This will create the docker image and name it `flood`.
 
 **To run a container based on the image:**
-1. `docker run -p 5055:5050 --name floodviz flood`. This will create a docker container from the `flood` image and name it `floodviz`. 
+1. `docker run -p 5055:5050 -t --name floodviz flood`. This will create a docker container from the `flood` image and name it `floodviz`.
```

Comment:
The `-t` tells docker to a terminal output for the container, without this it won't print "frozen" so it's hard to tell if it actually did freeze. When you start the container it runs the command at the end of the dockerfile, which is `python3 run.py --host 0.0.0.0 --freeze`.